bpf/bpf_lxc.c

```diff
@@ -1032,7 +1032,12 @@ ipv4_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label, __u8 *reason,
 
 	l4_off = l3_off + ipv4_hdrlen(ip4);
 	csum_l4_offset_and_flags(tuple.nexthdr, &csum_off);
-	is_fragment = ipv4_is_fragment(ip4);
+#if !defined IPV4_FRAGMENTS
```

Comment:
nit: `#ifndef ENABLE_IPV4_FRAGMENTS`; we usually have it as `ENABLE_` prefix for such configs

Throughout the code we also use `defined(...)`.

Reply:
There seems to be many `defined ...` too under the bpf/ directory. But sure, no problem! Will change to `#ifdef`/`#ifndef`. And I'll add the `ENABLE_` prefix.

Reply:
Right, the `defined ...` one is probably worth a bpf/ wide cleanup to make it consistent.

Reply:
Will add to WIP follow-up PR.